src/biogeophys/HydrologyNoDrainageMod.F90

```diff
@@ -272,6 +273,13 @@ subroutine HydrologyNoDrainage(bounds, &
        call water_inst%TracerConsistencyCheck(bounds, 'before main snow code in HydrologyNoDrainage')
     end if
 
+    do j = 1, nlevgrnd
```

Comment:
So is the idea that we would remove this calculation of h2osoi_vol_prehydrology as an answer changing tag when we are ready to do a larger answer change? Should we start planning for such an answering change tag?

Reply:
@ekluzek @dlawrenncar - Sorry, I wasn't very clear in describing this. My plan, if it's okay with both of you, is to make this additional answer change as part of the tag where I bring these changes in. So my intention was: after @ekluzek makes the ctsm5.1 fire tag, I would:
- Merge that into this branch
- Remove the temporary `h2osoi_vol_prehydrology`, so that the btran2 calculation depends on the updated version of h2osoi_vol in the time step
- Run test suite

This will change answers for all CN/BGC cases because btran2 will now use the later-in-timestep version of h2osoi_vol. My sense is that this shouldn't have a significant impact, but I can also hold off on making that answer change if you want to be conservative about answer changes right now.

Reply:
OK, makes sense.

Reply:
This was just a question. So resolving this.